apollo-compiler/src/test/graphql/com/example/fragment_friends_connection/FragmentFriendsConnectionExpected.java

```diff
@@ -1,15 +1,56 @@
 package com.example.fragment_friends_connection;
 
+import com.apollostack.api.GraphQLQuery;
+import java.lang.Override;
+import java.lang.String;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 import javax.annotation.Nullable;
 
-public interface FragmentFriendsConnection {
-  @Nullable Hero hero();
+public final class FragmentFriendsConnection implements GraphQLQuery<FragmentFriendsConnection.Data> {
```

Comment:
hmm I guess that makes sense since all the interface definition has moved to the `Data` interface 👍